jobserver/migrations/0029_convert_reportpublishrequest_report_to_an_fk.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 4.1.7 on 2023-05-03 10:22
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("jobserver", "0028_remove_publish_requests_approved_fields"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="releasefilepublishrequest",
+            name="snapshot",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="publish_requests",
+                to="jobserver.snapshot",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="reportpublishrequest",
+            name="report",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE,
```

Comment:
Just to confirm my understanding, `CASCADE` here means that if a PublishRequest happened to be deleted then the Report would be deleted too? Should this actually be set to `PROTECT` instead?

Reply:
Yes but otherway around!  If the target object of an FK (report here) is deleted then it's related objects (the publish requests) will also be deleted.